Physical review submission prep

drvinceknight opened the pull request.

marcharper commented on main.tex:
```diff
@@ -62,7 +62,22 @@
 interactive behaviour, having applications in biology, the study of human
-social behaviour, and many other domains. Since the introduction of
+social behaviour, and many other domains. A standard representation  of the game
+is given in equation~\ref{eqn:theipd}, the constraints ensure a non cooperative
```
where the constraints

marcharper commented on main.tex:
```diff
@@ -386,7 +405,10 @@ \section{Results: Numerical experiments}\label{sec:numerical-experiments}
-the always cooperating strategy Cooperator the \(\SSe\) is far from zero.
+the always cooperating strategy Cooperator the \(\SSe\) is far from zero. It is
+also clear that ZD-GTFT2 which is defined as a ZD strategy does not act
+extortionately which is evident by the fact that it does not rank highly according
```
remove second and third "which is" or simplify / breakup the sentence

marcharper commented on main.tex:
```diff
@@ -402,6 +424,30 @@ \section{Results: Numerical experiments}\label{sec:numerical-experiments}
     \label{fig:sserror_in_stewart_plotkin}
 \end{figure}
 
+In the next section a tournament with much larger number of strategies is
```
Next we consider a much larger tournament

marcharper commented on main.tex:
```diff
@@ -402,6 +424,30 @@ \section{Results: Numerical experiments}\label{sec:numerical-experiments}
 \end{figure}
 
+In the next section a tournament with much larger number of strategies is
+presented. As a final validation of the methodology proposed here,
```
of the proposed methodology

marcharper commented on main.tex:
```diff
@@ -402,6 +424,30 @@ \section{Results: Numerical experiments}\label{sec:numerical-experiments}
+presented. As a final validation of the methodology proposed here,
+Table~\ref{tbl:chi_versus_observed_chi_for_zd} shows the known values of
+\(\chi\) versus the measured values for all ZD strategies in the tournament. It
+is clear that the method accurately recovers \(\chi\) from the observed play of
```
Clearly the method...

marcharper commented on main.tex:
```diff
@@ -467,7 +513,9 @@ \section{Results: Numerical experiments}\label{sec:numerical-experiments}
         A similar conclusion to that of
         Figure~\ref{fig:sserror_in_stewart_plotkin} can be made: the strategies
-        that score highly have a negatively skewed \(\SSe\).}
+        that score highly have a negatively skewed \(\SSe\). The authors
```
Can we increase the resolution somehow? I feel like we shouldn't say "this isn't a high quality figure in print".

drvinceknight replied:
I believe (but please correct me) that the problem isn't resolution but just real estate of a printed page and needing/trying to include more than 200 names of strategies.

> I feel like we shouldn't say "this isn't a high quality figure in print".

I know exactly what you mean. It felt a wrong to write that but at the same time I felt it was not necessarily a bad thing to point out to readers/editors that it's a pity publications are somewhat still constrained by print.

*But* I've got a better suggestion (I believe):

The main thing we want from this figure is looking at the general trend of the scatter plot: we don't actually care about the particular strategies (just good to know that high rank by score have a high low skew etc). So what about this:

- In the paper, let us remove the names of the strategies all together so we just plot skew of SSE vs rank. Without needing to include the names we can have a larger axis label etc...
- We include as auxiliary material (so it would be available online) a pdf of the plot that includes the name: we could then make this as big as we want.
- Change the current caption to be something like: 
    ```
     \caption{\(\SSe\) for all strategies considered plotted against the rank according to score and wins.
    A similar conclusion to that of
    Figure~\ref{fig:sserror_in_stewart_plotkin} can be made: the strategies
    that score highly have a negatively skewed \(\SSe\). A version of this plot including the
    names of all strategies is available in the auxiliary materials.}
    ```

I'm happy to try any further suggestions.

marcharper replied:
That sounds reasonable to me.

drvinceknight replied:
Cool, I'll do that and put it in the style file. :+1:

marcharper commented on main.tex:
```diff
@@ -506,14 +554,15 @@ \subsection{Replicator Dynamics}
+stationary probabilities \(s\) of (\ref{eqn:replicator_dynamics}). Strategies
+that perform strongly according to equation (\ref{eqn:replicator_dynamics}) seem
+to be strategies that have a negative skew of \(\SSe\): indicating that they
+often have a high value of \(\SSe\) (ie do not act extortionately) but have a
```
i.e.

drvinceknight commented:
I've pushed cacc5b7 which I'm not convinced is necessary, just adds three examples of distributions of SSE... (Mainly reflecting on one of the reviewers who did not seem to understand the concept of a distribution). If anyone has strong opinions let me know.

drvinceknight commented:
I've modified the large figure and also updated the cover letter.

Looking at https://journals.aps.org/pre/authors#msprep it doesn't look like I need to do a lot of work to submit this, essentially just uploading the pdf (famous last words). If everyone is happy with this PR I'll aim to do that over next few days.

marcharper commented:
For Figure 3 can you update the x-axis to indicate that the ranks are increasing or decreasing? or perhaps plot versus the actual score or number of wins instead?

I know I've mentioned this before but I think the best fit lines on Figure 5 detract more than add.

I'm not sure Figure 6 adds much... I'm not opposed to leaving it in but if it's just to appease one former reviewer than I'd lean toward leaving it out.

drvinceknight commented:
> For Figure 3 can you update the x-axis to indicate that the ranks are increasing or decreasing? or perhaps plot versus the actual score or number of wins instead?

I believe plotting vs the actual score and number of wins makes the plot unclear but that might have been because we were keen to include the names... I'll see how it looks but if we keep ranks I'll indicate that ranks are increasing.

> 
> I know I've mentioned this before but I think the best fit lines on Figure 5 detract more than add.

I was thinking the same. I'll take them out.

> 
> I'm not sure Figure 6 adds much... I'm not opposed to leaving it in but if it's just to appease one former reviewer than I'd lean toward leaving it out.

I know what you but I do think it's nice to include an example of the distributions.

Thanks for taking the time to go through again :+1:

marcharper commented:
SGTM

drvinceknight commented:
I've updated @marcharper (sorry for being so slow with all this :+1:)
